timesketch/lib/analyzers/contrib/README.md

```diff
@@ -0,0 +1,26 @@
+## Community contributed analyzers
```

Comment:
I opt that we move some of that to https://github.com/google/timesketch/blob/master/docs/developers/analyzer-development.md and then link from here to the docs folder.

Reply:
👍